oceans/filters.py

```diff
@@ -608,14 +605,25 @@ def pl33tn(x, dt=1.0, T=33.0, mode="valid"):
     """
 
     import cf_xarray  # noqa: F401
+    import pandas as pd
     import xarray as xr
 
-    if isinstance(x, xr.Dataset):
-        raise TypeError("Input a DataArray not a Dataset.")
+    if isinstance(x, xr.Dataset | pd.DataFrame):
+        raise TypeError("Input a DataArray not a Dataset, or a Series not a DataFrame.")
 
+    if isinstance(x, pd.Series) and not isinstance(
+        x.index,
+        pd.core.indexes.datetimes.DatetimeIndex,
+    ):
+        raise TypeError("Input Series needs to have parsed datetime indices.")
+
+    # find dt in units of hours
     if isinstance(x, xr.DataArray):
-        # find dt in units of hours
-        dt = (x.cf["T"][1] - x.cf["T"][0]) * 1e-9 / 3600
+        dt = (x.cf["T"][1] - x.cf["T"][0]) / np.timedelta64(
+            360_000_0000_000,
```

Comment:
This is the same as 360000000000 but more readable.
```suggestion
            360_000_000_000,
```